src/python/finalize/cli/main.go

```diff
@@ -1,15 +1,16 @@
 package main
 
 import (
-	"github.com/cloudfoundry/python-buildpack/src/python/finalize"
-	_ "github.com/cloudfoundry/python-buildpack/src/python/hooks"
-	"github.com/cloudfoundry/python-buildpack/src/python/pyfinder"
-	"github.com/cloudfoundry/python-buildpack/src/python/requirements"
 	"io"
 	"io/ioutil"
 	"os"
 	"time"
 
+	"github.com/cloudfoundry/python-buildpack/src/python/finalize"
```

Comment:
Super minor, but typically this "first-party" group would be swapped with the third-party group below (anything not standard go and not part of this repo).

Just fyi, not worth changing :)